app/src/main/java/govind/iiitl/app/models/Uploads.java

```diff
@@ -0,0 +1,30 @@
+package govind.iiitl.app.models;
+
+public class Uploads {
+
+    public String name;
```

Comment:
make private

Reply:
@GOVINDDIXIT  I have checked this ... But still it's causing error and this has been resolved by using a pojo class . But the items are not still visible only their count can be seen

Reply:
@refactor-droidyy Don't worry I will fix it 😃